src/diffusers/pipelines/animatediff/pipeline_animatediff.py

```diff
@@ -79,6 +76,12 @@ def tensor2vid(video: torch.Tensor, processor, output_type="np"):
 
         outputs.append(batch_output)
 
+    if output_type == "np":
+        outputs = np.stack(outputs)
+
+    if output_type == "pt":
+        outputs = torch.stack(outputs)
```

Comment:
```suggestion
    if output_type == "np":
        outputs = np.stack(outputs)
    elif output_type == "pt":
        outputs = torch.stack(outputs)
    else:
        raise ValueError(f"{output_type} does not exist. Please choose one of ['np', 'pt']")
```

Or should we also allow returning a list of PIL images?

Reply:
If yes we could maybe change:
```diff
-     else:
+     elif not output_type == "pil":
```

Reply:
We do support returning a PIL list as well. Will update.